packages/babel-parser/test/fixtures/jsx/errors/_no-plugin-ts-type-param/input.js

```diff
@@ -0,0 +1 @@
+<div>() => {}
```

Comment:
This case is parsed as flow type annotation. Can you add an extra test but `flow` is _not_ enabled?

Reply:
Can do :)